iina/Extensions.swift

```diff
@@ -290,6 +299,12 @@ extension FloatingPoint {
       return self
     }
   }
+
+  /// Formats as String, rounding the number to 2 digits after the decimal
+  var stringMaxFrac2: String {
```

Comment:
I suggest using a name like `stringWithMaxFractionDigit2` for better readability.

Reply:
Ah yeah. For utility functions like this which might be commonly used, I worry about them getting too long. But, just semantics. Done